src/lib/containers/dataaccess/RejectDataAccessRequestModal.tsx

```diff
@@ -0,0 +1,422 @@
+const DEFAULT_MESSAGE_APPEND =
+  '\nIf you have questions, do not respond to this email address. Instead, reply to:\nact@sagebionetworks.org'
+
+/**
+ * Renders a rejection reason category and checkboxes for each reason in the category
+ */
+function RejectionCategory(props: RejectionCategoryProps) {
+  const {
+    category,
+    rows,
+    selectedRowIds,
+    setSelectedRowIds,
+    rejectionReasonFormTextIndex,
+  } = props
+  const [isExpanded, setIsExpanded] = React.useState(false)
+
+  return (
+    <>
+      <Typography
+        variant="body1"
+        onClick={() => setIsExpanded(!isExpanded)}
+        sx={{ fontWeight: 700, cursor: 'pointer', my: 1 }}
+      >
+        <IconSvg
+          icon={isExpanded ? 'expandMore' : 'chevronRight'}
+          sx={{ color: 'grey.700' }}
+          wrap={false}
+        />
+        {category}
+      </Typography>
+      <Collapse in={isExpanded}>
+        <Stack sx={{ ml: 3 }}>
+          {(rows ?? []).map(row => (
+            <FormControlLabel
+              key={row.rowId}
+              control={
+                <Checkbox
+                  checked={selectedRowIds.has(row.rowId!)}
+                  size={'small'}
+                  onChange={event => {
+                    if (event.target.checked) {
+                      setSelectedRowIds(selectedRowIds.add(row.rowId!))
+                    } else {
+                      setSelectedRowIds(selectedRowIds.remove(row.rowId!))
+                    }
+                  }}
+                />
+              }
+              label={
+                <Typography variant={'smallText1'}>
+                  {row.values[rejectionReasonFormTextIndex]}
+                </Typography>
+              }
+            />
+          ))}
+        </Stack>
+      </Collapse>
+    </>
+  )
+}
+
+/**
+ * Renders a form for selecting individual rejection reasons grouped by category.
+ * The rejection reason data comes from the table whose data is fetched in the tableQuery prop.
+ */
+function SelectRejectionReasonsForm(props: SelectRejectionReasonsFormProps) {
+  const { tableQuery, selectedRowIds, setSelectedRowIds } = props
+  const { data: tableData, isLoading, error } = tableQuery
+  const categoryIndex = tableData?.queryResult?.queryResults.headers.findIndex(
+    header => header.name.toLowerCase() === CATEGORY_COLUMN_NAME,
+  )
+  const rejectionReasonFormTextIndex =
+    tableData?.queryResult?.queryResults.headers.findIndex(
+      header =>
+        header.name.toLowerCase() === REJECTION_REASON_FORM_TEXT_COLUMN_NAME,
+    )
+
+  const rowsGroupedByCategory =
+    tableData &&
+    tableData.queryResult &&
+    tableData.queryResult.queryResults.rows.reduce(
+      (acc: Record<string, Row[]>, row) => {
+        const category: string = row.values[categoryIndex!]!
+        acc[category] = [...(acc[category] || []), row]
+        return acc
+      },
+      {},
+    )
+  return (
+    <>
+      <Typography variant="headline3" gutterBottom>
```

Comment:
The `gutterBottom` prop adds a standardized bottom margin to the Typography component.